base/libgit2/types.jl

```diff
@@ -1119,14 +1119,29 @@ struct CachedCredentials <: AbstractCredentials
     CachedCredentials() = new(Dict{String,AbstractCredentials}())
 end
 
-"Obtain the cached credentials for the given host+protocol (credid), or return and store the default if not found"
-get_creds!(collection::CachedCredentials, credid, default) = get!(collection.cred, credid, default)
```

Comment:
Should probably deprecate `get_creds!` properly.

Reply:
:+1:

Reply:
Added a proper deprecation for this. It required adding [`Base.get!(cache::CachedCredentials, cred_id, default)`](https://github.com/JuliaLang/julia/pull/23711/files#diff-13a2010bd5b11c9f5d12b75cfba76bc9R1124). Additionally, I switched the test/misc.jl tests to use `get!`.